Sources/FluentUI_common/Core/Extensions/NSColor+Extensions.swift

```diff
@@ -51,6 +51,12 @@ extension NSColor {
         return resolvedColorValue(appearance: NSAppearance(named: .darkAqua))
     }
 
+    convenience init(dynamicColor: DynamicColor) {
+        let colorResolver = { $0 != nil ? NSColor($0!) : nil }
+        self.init(light: NSColor(dynamicColor.light),
+                  dark: colorResolver(dynamicColor.dark))
+    }
```

Comment:
Maybe add a comment that the purpose of color resolver is to ensure proper fallback behavior? IDK, your choice

Reply:
Sure -- it's copied from the UIColor equivalent of this file, so I can comment both.